index.js

```diff
@@ -240,6 +240,17 @@ function inferTypeByKeyword (schema) {
   for (var keyword of numberKeywords) {
     if (keyword in schema) return 'number'
   }
+
+  if (schema.const) {
```

Comment:
I don't understand why we need to get a schema type if it's a const. Shouldn't we just substitute the value of the schema.const?

Reply:
You right, I forgot to remove that